techniques/general/G220-display-values.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+	<head>
+		<title>Displaying the previously entered value</title>
+		<link rel="stylesheet" type="text/css" href="../../css/editors.css"/>
+	</head>
+	<body>
+		<h1>Displaying the previously entered value</h1>
+		<section id="meta">
+			<h2>Metadata</h2>
+			<p class="instructions">Provide information below to help editors associate the technique properly. Contents of the meta section are not output by the processor.</p>
+			<p id="sc">3.3.8 Redundant entry</p>
+			<p id="type">Sufficient</p>
+		</section>
+		<section id="applicability">
+			<h2>Applicability</h2>
+			<p class="instructions">Any technology that supports scripting.</p>
+		</section>
+		<section id="description">
+			<h2>Description</h2>
+			<p>The objective of this technique is to display information that was previously provided by the user or by the system rather than requiring the user to remember the information from a previous step.  This improves success for individuals with cognitive and learning disabilities and memory impairments.</p>
+
+			<p>This technique best applies when information is almost always the same as information previously entered or provided.</p>
+				
+		</section>
+		<section id="examples">
+			<h2>Examples</h2>
+			<section class="example">
+				<h3>Display the information from a previous step</h3>
+				<p>During a two step process for setting up a business, the first step provides a business identification number after the user enters a name and address. The second step requires creating a user profile. The interface allows the user to select the business they just set up with all the associated information.</p>
+			</section>
+		</section>
+		<section id="tests">
+			<h2>Tests</h2>
+			<section class="test-procedure">
+				<h3>Procedure</h3>
+				<p>For each input in a process that requires information to be entered:</p>
+				<ol>
+					<li>Check if that information has already been requested during the process;</li>
+					<li>Check that the information entered previously is displayed on the page.</li>
```

Comment:
This technique is the same as the previous one. The only difference in the tests is whether the information is "already prepopulated in the fields" or "displayed on the page". Anything that passes the previous technique will pass this one. Do we need both?

Reply:
I've combined them... wish I'd seen that comment first!